cmd/vicadmin/server.go

```diff
@@ -255,7 +255,8 @@ func (s *server) tailFiles(res http.ResponseWriter, req *http.Request, names []s
 func (s *server) index(res http.ResponseWriter, req *http.Request) {
 	defer trace.End(trace.Begin(""))
 	ctx := context.Background()
-	v := vicadmin.NewValidator(ctx, vchConfig)
+	sess, err := client()
```

Comment:
install/validator.NewValidator actually takes the input.Data (as specified on the command-line) and constructs a vch config spec from that.  In this case I already have the VCH config and only need to run certain segments of the install validation.